draft-iab-use-it-or-lose-it.md

```diff
@@ -216,11 +216,15 @@ extension were introduced.
 One of the key challenges in deploying new features is ensuring compatibility
 with all actors that could be involved in the protocol.  Even the most
 superficially simple protocols can often involve more actors than is immediately
-apparent.  Even for a two-party protocol, protocol elements can be passed on to
-other entities in ways that can affect protocol operation.
+apparent.
 
-Protocols that are intermediated need to consider the effect that deploying an
-extension might have on a middlebox.
+Actions that might be taken by middleboxes in response to a change in a protocol
+and the effect that might have on the operation of the protocol require careful
+consideration when designing for extensibility.
+
+Changes might affect entites with indirect interactions with a protocol. How
+protocol elements are handled in applications or other interconnected systems
+might need to be considered when making changes.
 
```

Comment:
Thanks, that's much more clear. I guess  "indirect interaction" is still a bit vague and might not be clear to all readers. Maybe something like "entities which are not an explicit communication endpoint" is a bit clearer? Or you just add an example, e.g. "like middleboxes or firewalls".